### PR TITLE
Update en-replace.ru.pairs

### PR DESCRIPTION
**Pull request by HSteinway**
Can we do this? It haunts me) In Russian we have different inflexions almost for EVERY number:
20 фраз
21 фраза
22 фразы
..
25 фраз

### CODE REVIEW

File: translated-pairs/en-replace.ru.pairs [hunk 1]
```diff
@@ -300,20 +300,20 @@ Total number of evaluated code phrases
...
+Кол-во фраз в решении:
 
 Run size:
-Размер выполнения:
+Общее кол-во фраз:
 
 #### right half `{}` is replaced by the score
 {} ms
-{} мс
+{}
```
Comment (HSteinway) [inline translated-pairs/en-replace.ru.pairs:313]: can we get rid of this?

File: translated-pairs/en-replace.ru.pairs [hunk 2]
```diff
@@ -300,20 +300,20 @@ Total number of evaluated code phrases
...
+Общее кол-во фраз:
 
 #### right half `{}` is replaced by the score
 {} ms
-{} мс
+{} 
 
 {} phrases
-{} (фразы)
+{}
```
Comment (HSteinway) [inline translated-pairs/en-replace.ru.pairs:316]: and this

**Comment by alexheretic** [comment]
> Can we do this? It haunts me) In Russian we have different inflexions almost for EVERY number:
> 20 фраз
> 21 фраза
> 22 фразы
> ..
> 25 фраз

This isn't possible with the current translation framework, but I can add it in. Let me know whatever you need to make the translation better and I'll try to add it!

**Comment by HSteinway** [comment]
No, no, I mean can we get rid of "мс" и "фразы" completely only in Russian version?

**Comment by alexheretic** [comment]
>  I mean can we get rid of "мс" и "фразы" completely only in Russian version?

Absolutely, if you think it's better that way. This PR should work.

**Comment by HSteinway** [comment]
Then it's ready to merge. =)

**Comment by HSteinway** [comment]
Anyway, I'll play on Russian for some time and then I'll make one thousand more edits)

**Comment by alexheretic** [comment]
@HSteinway if you want them let me send you a couple of extra game keys as a thank you (email me at alex@roboinstruct.us). Maybe you can give them away?

**Comment by HSteinway** [comment]
I'd better force someone to buy it on Steam) Thank you anyway, I very appreciate this. Programming was my childish dream, but I chose another way... I'm just glad to help. Consider it as a part of the game for me. =)

**Comment by alexheretic** [comment]
@HSteinway I've made a pre-release build available on steam. Right-click properties / betas / select `test - Pre-release testing`. This is a preview of the next build with the currently translation.

**Comment by HSteinway** [comment]
Looks good! Some edits incoming) Questions:
Where is 'Robot fell' string? (my favourite one)
What is 'Step over'? (interface, didn't find ingame)

**Comment by HSteinway** [comment]
Russian phrases are loo long, made them shorter. I found Step Over, nvm. Translated the levels names, did my best, but comments are very welcomed (especially about the later ones, where I haven't played yet).

**Pull request merged**